presto-docs/src/main/sphinx/release/release-0.124.rst

```diff
@@ -6,6 +6,8 @@ General Changes
 ---------------
 
 * The :func:`approx_percentile` aggregation now also accepts an array of percentages.
+* Fix issue where invalid plans are generated for queries with multiple aggregations that require input values to be cast in different ways.
```

Comment:
How about:

Fix issue where aggregation queries that require inputs to be implicitly coerced to different types produce invalid plans.